modules/bmo-configuring-network-data.adoc

```diff
@@ -0,0 +1,54 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-configuring-networkdata_{context}"]
+= Configuring networkData
```

Comment:
```suggestion
= Controlling the network configuration of the bare-metal host at creation time
```
ok, but what is "creation time"? creation of the CR or the bare-metal host config, er?